### PR TITLE
Security Adapter Functionality for Exorelay/Exoservice

### PR DESCRIPTION
**Pull request by mcharawi**

### CODE REVIEW

**Comment by charlierudolph** [comment]
We also need this for the javascript implementations. Can we add that to this PR as well?

**Comment by mcharawi** [comment]
@charlierudolph added

File: javascript/exorelay/features/outgoing-replies.feature [hunk 1]
```diff
@@ -114,3 +114,33 @@ Feature: Sending outgoing replies to incoming messages
       activity-id: '456'
       auth: '1'
       """
+
+
+  Scenario: sending a reply with security flag
```
Comment (charlierudolph) [inline javascript/exorelay/features/outgoing-replies.feature]: This sounds like the reply is setting a flag whereas the flag is on the incoming message. thoughts on updating this to "sending a reply when the incoming message has the security flag"

Comment (mcharawi) [reply javascript/exorelay/features/outgoing-replies.feature]: 👍

**Pull request merged**